src/pyfmi/examples/fmi20_bouncing_ball_native.py

```diff
@@ -104,11 +104,13 @@ def run_demo(with_plots=True):
         event_ind_new = bouncing_fmu.get_event_indicators()
         
         #Inform the model about an accepted step and check for step events
-        step_event = bouncing_fmu.completed_integrator_step()
+        step_event, terminate = bouncing_fmu.completed_integrator_step()
+        if terminate: 
+            return
         
         #Check for time and state events
         time_event  = abs(time-Tnext) <= 1.e-10
-        state_event = True if True in ((event_ind_new>0.0) != (event_ind>0.0)) else False
+        state_event = True in ((event_ind_new>0.0) != (event_ind>0.0))

```

Comment:
I might be missing something but isn't it enough to write `state_event = (event_ind_new>0.0) != (event_ind>0.0)`?

Reply:
In general, no, since event_ind(_new) is an array of all event indicators and this checks if any changed sign.

It would work in this particular example though, since there is only a single event indicator. I'd still keep the general one though.

Reply:
I see, thank you for the clarification!